pom.xml

```diff
@@ -16,19 +16,22 @@
         <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
-<dependencies>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-android</artifactId>
+            <version>3.2.4</version>
+            <type>pom</type>
+        </dependency>
```

Comment:
This is wrong dependencies.
Use this one - https://mvnrepository.com/artifact/org.mockito/mockito-core/2.10.0